openshift-templates/README.adoc

```diff
@@ -0,0 +1,5 @@
+Coming soon. In between:
```

Comment:
Slightly confused by the text - do you mean "Coming soon, but in the meantime:" ?